pkg/loader/fileloader.go

```diff
@@ -53,6 +53,9 @@ func (l *fileLoader) Root() string {
 // Example: "/home/seans/project" or "/home/seans/project/"
 // NOT "/home/seans/project/file.yaml".
 func (l *fileLoader) New(newRoot string) (Loader, error) {
+	if isRepoUrl(newRoot) {
+		return newGithubLoader(newRoot, l.fSys)
+	}
```

Comment:
NewLoader does this (it doesn't look right here).

Reply:
There is slight difference between `New` and `NewLoader`. `New` creates a new fileLoader whose root is a relative path in current fileloader. `NewLoader` is more like creating a root loader, where we don't have an existing fileloader.